proposals/0269-New-vehicle-data-ClimateData.md

```diff
@@ -23,19 +23,23 @@ We need to add `ClimateData` for `GetVehicleData`, `SubscribeVehicleData`, `Unsu
 
 ```xml	
 <element name="VEHICLEDATA_CLIMATEDATA" since="X.x"/>
-<element name="VEHICLEDATA_EXTERNTEMP" until="X.x"/>
+<element name="VEHICLEDATA_EXTERNTEMP" until="X.x">
+	<history>
+		<element name="VEHICLEDATA_EXTERNTEMP" />
+	</history>
+</element>
```

Comment:
Please update this history tag to proper formatting, including since and until values. See example: https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml#L2250

Reply:
Done